lib/modules/manager/gitlabci/__fixtures__/gitlab-ci.8.yaml

```diff
@@ -0,0 +1,33 @@
+job1:
```

Comment:
Can't we simply add a new value to existing test instead of crfeating this huge new test?